src/components/ParentalLineModal.tsx

```diff
@@ -55,24 +57,28 @@ const ParentalLineModal = (props: ParentalLineModalProps) => {
                 className={modal}
                 footer={
                     <div style={{ textAlign: "center" }}>
-                        <Button
-                            type="primary"
-                            style={{ width: 480, border: "2px solid #003075" }}
+                        <DarkBlueHoverButton
+                            style={{
+                                width: 480,
+                                borderWidth: "2px",
+                                backgroundColor: "white",
+                            }}
                             href="https://www.allencell.org/cell-catalog.html"
                             target="_blank"
                             className={actionButton}
                         >
-                            <Flex justify="flex-end" gap={142}>
-                                More information
-                                <Icon
-                                    component={LinkOut}
-                                    style={{
-                                        color: "transparent",
-                                        fontSize: "24px",
-                                    }}
-                                />
+                            <Flex
+                                justify="flex-end"
+                                gap={110}
+                                style={{
+                                    width: "100%",
+                                    fontWeight: 600,
+                                }}
+                            >
+                                Go to Parental Line Page
```

Comment:
Nit: No idea if this is on the design, but slightly prefer having the rest of this be lowercase after Go?